### DISCUSSION TITLE
Assistance required to run M365 assessments

### DISCUSSION THREAD

**Discussion by isachd** [silverhack/monkey365#58] [General]
I am using a M365 developer account to test the tool, hence it doesn't have a running Azure subscription. 

I ran the following command, 

 $param = @{
Instance = 'Microsoft365';
Analysis = 'ExchangeOnline','Microsoft365','MicrosoftTeams','SharePointOnline'     ;
PromptBehavior = 'SelectAccount';
IncludeAzureAD = $true;
ExportTo = 'HTML';
}
$assets = Invoke-Monkey365 @param

and was able to successfully get the results for  Azure AD, IAM, Exchange Online and Microsoft Teams. 

What would be the right command to also get the results to also include for Microsoft Forms, Security and Compliance?

**Comment by silverhack** [comment]
Hi @isachd, 

You can find documentation regarding Microsoft365 options [here](https://silverhack.github.io/monkey365/configuration/microsoft365-options/)
